en/core-api/direct-debit.md

```diff
@@ -1,15 +1,15 @@
-# Direct Debit Transfer Integration
+# Direct Debit Payment Integration
 <hr>
-One of the payment method offered by Midtrans is Direct Debit. By using this payment method, customers will have the option to make a payment via bank website and Midtrans will send real time notification when the customer complete the payment.
+<i>Direct Debit</i> is one of the payment methods offered by Midtrans. Using this payment method, the customer can authorize you to withdraw funds from the bank account. It is usually done for recurring payments, where the payment amount varies from one payment to another. This payment method can also be used for irregular payments such as for mail order transactions or at a point of sale (POS). Midtrans sends real-time notifications when the customer completes the payment.
```

Comment:
> Using this payment method, the customer can authorize you to withdraw funds from the bank account. It is usually done for recurring payments, where the payment amount varies from one payment to another. This payment method can also be used for irregular payments such as for mail order transactions or at a point of sale (POS). Midtrans sends real-time notifications when the customer completes the payment.

This description may cause misunderstanding, this is not `withdraw`, because `withdraw` can also means payout. And it is not quite for `recurring payments`. 

> also be used for irregular payments such as for mail order transactions or at a point of sale (POS)
We don't need to explain this.

This payment method is just about letting customer pay using their E-Banking Debit Account.